pkg/ibmvpc/examples/out/explain_out/LBToIksNode_all_vpcs_explain_debug.txt

```diff
@@ -0,0 +1,360 @@
+
+Allowed connections from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca[LB private IP][192.168.40.6] to iks-node[192.168.4.4]: protocol: TCP,UDP dst-ports: 30000-32767
+
+Path:
+	kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca[LB private IP][192.168.40.6] -> security group kube-r006-d7cfb31a-1d4b-40c8-83df-ce2e6f8f2e57 -> ky-test-edge-subnet-3 -> network ACL ky-test-edge-acl -> 
+	network ACL ky-test-private-2-others-acl -> ky-test-private-subnet-2 -> security group[kube-clusterid:1, ky-test-default-sg] -> iks-node[192.168.4.4]
+
+
+Details:
+~~~~~~~~
+Path is enabled; The relevant rules are:
+	Load Balancer:
+		kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca[LoadBalancer] allows connection to iks-node[192.168.4.4] which is one of its pool members
+
+	Egress:
+		security group kube-r006-d7cfb31a-1d4b-40c8-83df-ce2e6f8f2e57 allows connection with the following allow rules
+			index: 0, direction: outbound,  conns: protocol: tcp,  dstPorts: 30000-32767, remote: 0.0.0.0/0, local: 0.0.0.0/0
+			index: 1, direction: outbound,  conns: protocol: udp,  dstPorts: 30000-32767, remote: 0.0.0.0/0, local: 0.0.0.0/0
+		network ACL ky-test-edge-acl allows connection with the following allow rules
+			index: 1, direction: outbound , src: 0.0.0.0/0 , dst: 0.0.0.0/0, conn: all, action: allow
+
+	Ingress:
+		network ACL ky-test-private-2-others-acl allows connection with the following allow rules
+			index: 0, direction: inbound , src: 192.168.32.0/20 , dst: 0.0.0.0/0, conn: all, action: allow
+		security group kube-clusterid:1 allows connection with the following allow rules
+			index: 3, direction: inbound,  conns: protocol: tcp,  dstPorts: 30000-32767, remote: 0.0.0.0/0, local: 0.0.0.0/0
+			index: 4, direction: inbound,  conns: protocol: udp,  dstPorts: 30000-32767, remote: 0.0.0.0/0, local: 0.0.0.0/0
+			index: 6, direction: inbound,  conns: protocol: icmp,  icmpType: protocol: ICMP icmp-type: 8, remote: 0.0.0.0/0, local: 0.0.0.0/0
+		security group ky-test-default-sg allows connection with the following allow rules
+			index: 2, direction: inbound,  conns: protocol: all, remote: 0.0.0.0/0, local: 0.0.0.0/0
+
+TCP response is enabled; The relevant rules are:
+	Egress:
+		network ACL ky-test-private-2-others-acl allows connection with the following allow rules
+			index: 4, direction: outbound , src: 0.0.0.0/0 , dst: 192.168.32.0/20, conn: all, action: allow
+
+	Ingress:
+		network ACL ky-test-edge-acl allows connection with the following allow rules
+			index: 0, direction: inbound , src: 0.0.0.0/0 , dst: 0.0.0.0/0, conn: all, action: allow
+
+------------------------------------------------------------------------------------------------------------------------
+
+Allowed connections from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca[Potential LB private IP][192.168.0.0/22] to iks-node[192.168.4.4]: protocol: TCP,UDP dst-ports: 30000-32767
```

Comment:
What is the logic for the creation of a potential LB private IP? If this is an endpoint that we invented I'd expect to see a nicer and shorter name for it, maybe concatenate the LB name with `_ip_` and the private IP index?

Reply:
its <lb name>[<end point type>][<ip range>]. the ip range is the rang that we create the pip for.
yes, in this example the name is not pretty.